python/hail/tests/test_methods.py

```diff
@@ -909,16 +909,6 @@ def test_pca(self):
 
         _, _, loadings = hl.pca(mt.GT.n_alt_alleles(), k=2, compute_loadings=False)
         self.assertEqual(loadings, None)
-        
```

Comment:
It seems bad to completely remove a test?

Reply:
we briefly discussed this yesterday. This test was created as part of our "if you fix a bug, make a test" policy, and fixed a bug caused by PCA doing its own AST/eval stuff. PCA now calls the base expr API, so this test is totally unnecessary. The expr stuff should be tested in its own modules, and PCA should be tested for its own method correctness (https://github.com/hail-is/hail/pull/3613), but we shouldn't need to test whether all expressions are valid input for PCA (doing this everywhere would be insane!)

Reply:
also, I don't think this is even a valid test given that we don't guarantee determinism

Reply:
Ok! In the future, let's try to keep these changes as separate PRs with a description as to why we are removing it. I'll approve now! 😀